controllers/snackController.js

```diff
@@ -16,21 +21,32 @@ snacks.get("/", async (req, res) => {
 snacks.post("/", async (req, res) => {
   try {
     const snack = await createSnack(req.body);
-    res.json(snack);
+    res.status(201).json(snack);
```

Comment:
This is nitpicky but aren't 201 status codes for when a resource is created with POST requests?